- fluxo_name: extracao_jira_sprint
  horario: '13:24'
  data: 25/09/2025
  status: Falha
  mensagem: 'HTTPConnectionPool(host=''massai-api'', port=8000): Max retries exceeded
    with url: /run_jira/ (Caused by NewConnectionError(''<urllib3.connection.HTTPConnection
    object at 0x7fd0eb22fb10>: Failed to establish a new connection: [Errno 111] Connection
    refused''))'
  endpoint: /run_jira/
  api_url: http://massai-api:8000
- fluxo_name: extracao_zephyr_diaria
  horario: '13:25'
  data: 25/09/2025
  status: Falha
  mensagem: 'HTTPConnectionPool(host=''massai-api'', port=8000): Max retries exceeded
    with url: /run_zephyr/ (Caused by NewConnectionError(''<urllib3.connection.HTTPConnection
    object at 0x7fd0eb238a90>: Failed to establish a new connection: [Errno 111] Connection
    refused''))'
  endpoint: /run_zephyr/
  api_url: http://massai-api:8000
